assets/src/dashboard/icons/lock.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="15px" height="15px" viewBox="0 0 15 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 60.1 (88133) - https://sketch.com -->
+    <title>Artboard</title>
+    <desc>Created with Sketch.</desc>
```

Comment:
Where is this icon coming from? 🤔

Reply:
Copied and pasted from https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/icons/lock.svg

Reply:
Let's use `lock_closed.svg` from the design system instead of duplicating files.

Reply:
I used this icon, but had to do some series hacking around to make it look good, as the icon has one space around it